Source/Orts.Simulation/Common/Scripting/TrainControlSystem.cs

```diff
@@ -143,10 +143,6 @@ public abstract class TrainControlSystem : AbstractScriptClass
         /// </summary>
         public Func<float> TrainLengthM;
         /// <summary>
-        /// Train's actual absolute speed.
-        /// </summary>
-        public Func<float> SpeedMpS;
-        /// <summary>
```

Comment:
Is that intentional ?

Reply:
Yes, I moved it to AbstractScriptClass. Does this cause a problem?